extensions/aerial-photo/schema.json

```diff
@@ -90,6 +93,70 @@
         "aerial-photo:anomalies": {
           "title": "Image anomalies",
           "type": "string"
+        },
+        "summaries": {
+          "type": "object",
+          "required": ["aerial-photo:run", "aerial-photo:sequence_number"],
+          "properties": {
+            "aerial-photo:run": {
+              "type": "array",
+              "minLength": 1,
+              "minItems": 1,
+              "items": {
+                "type": "string"
+              }
+            },
+            "aerial-photo:sequence_number": {
+              "type": "object",
+              "required": ["minimum", "maximum"],
+              "properties": {
+                "minimum": {
+                  "title": "lowest sequence number",
+                  "type": "integer"
+                },
+                "maximum": {
+                  "title": "highest sequence number",
+                  "type": "integer"
+                }
+              }
+            },
+            "aerial-photo:altitude": {
+              "type": "object",
+              "required": ["minimum", "maximum"],
+              "properties": {
+                "minimum": {
+                  "title": "lowest altitude",
+                  "type": "integer"
+                },
+                "maximum": {
+                  "title": "highest altitude",
+                  "type": "integer"
+                }
+              }
+            },
+            "aerial-photo:scale": {
+              "type": "object",
+              "required": ["minimum", "maximum"],
+              "properties": {
+                "minimum": {
+                  "title": "minimum scale",
+                  "type": "integer"
+                },
+                "maximum": {
+                  "title": "maximum scale",
+                  "type": "integer"
+                }
+              }
+            },
+            "aerial-photo:anomalies": {
+              "type": "array",
+              "minLength": 1,
```

Comment:
`minLength` is for strings, I think. Did you mean to put it inside `items`? You [should only need `minItems`](https://json-schema.org/understanding-json-schema/reference/array.html#length) here. Ditto for other array properties.

Reply:
Also what happens when there are no annomilies do we just have `aerial-photo:anomalies: [null]` ?

Reply:
I think it was decided that if we have nothing for the field and its not a compulsory item field it wont get included in the summaries. @dwsilk @amfage is that right?

Reply:
Yes, if the field isn't required and doesn't have a value the field isn't included (in the item properties or the summaries).

Reply:
Yes what @amfage said.. only if it was mandatory but accepted null, then that would result in a summary with null in it.